Snakefile

```diff
@@ -18,6 +17,7 @@ COSMIC_EMAIL = config.get('cosmic_email')
 COSMIC_PASSWORD = config.get('cosmic_password')
 USE_COSMIC = COSMIC_EMAIL or COSMIC_PASSWORD
```

Comment:
is COSMIC_EMAIL now defined elsewhere?

Reply:
no it hasn't changed. it is still defined in line 16